.gitignore

```diff
@@ -104,5 +104,8 @@ ENV/
 /.idea
 *.iml
 
+# Pytest cache
+.pytest_cache
```

Comment:
This doesn't seem to be working? There are cache files just below!

Reply:
Seems to be from before I have deleted it.